api/metadata_endpoints.py

```diff
@@ -30,11 +37,40 @@ def get_landing_page(request):
     )
 
 
-def get_collection_metadata(request) -> Collection:
+async def get_collection_metadata(request, is_self) -> Collection:
+    # TODO: Try to remove/lower duplication with /locations endpoint
```

Comment:
Due to the `platform_parameters` and `platform_coordinates` in `\locations`, I think that this is challenging and can negatively impact the readability of the code.